src/actions/index.js

```diff
@@ -1,4 +1,4 @@
-import { FETCH_USER_INFO } from './constants';
+import { FETCH_USER_INFO } from '../types';
```

Comment:
`import { FETCH_USER_INFO } from '../userActions';` - rename that file from `types` to `userActions`.

Reply:
We have all the action types in the index.js file in the types directory.